test/test-TabTracker.js

```diff
@@ -93,7 +93,7 @@ exports.test_TabTracker_reactivating = function*(assert) {
   let tabClosedPromise = new Promise(resolve => {
     for (let i in openTabs) {
       openTabs[i].close(() => {
-        if (i == openTabs.length - 1) {
+        if (Number(i) === openTabs.length - 1) {
```

Comment:
Turns out, `i` was a String here, so `==` was doing the casting thing. I had to explicitly wrap in `Number()` otherwise tests were a failing.